templates/Includes/RealMeLoginButton.ss

```diff
@@ -0,0 +1 @@
+<span class="realme_button_padding">$Label<span class="realme_icon_new_window"></span> <span class="realme_icon_padlock"></span>
```

Comment:
I was going to say, you should have just used the `<%t %>` in the template to avoid having the extra php code to inject the string.